### PR TITLE
fix to integrate config builder docstrings to entropylab document…

### PR DESCRIPTION
**Pull request by SatyaBade12**
Fix to make py-qua-tools compatible with mkdocs

### CODE REVIEW

File: qualang_tools/bakery/randomized_benchmark_c1.py [hunk 1]
```diff
@@ -29,31 +29,32 @@
...
-        18, 18, 15, 19, 14, 16, 22, 17, 23, 21, 20, 13, 12, 11, 10, 3, 1, 4, 6, 0, 2, 9, 8, 5, 7,
-        19, 19, 14, 18, 15, 23, 17, 22, 16, 13, 12, 21, 20, 9, 8, 1, 3, 7, 5, 2, 0, 11, 10, 6, 4,
-        20, 20, 21, 13, 12, 19, 18, 15, 14, 17, 22, 16, 23, 3, 2, 7, 6, 10, 8, 5, 4, 0, 1, 9, 11,
-        21, 21, 20, 12, 13, 15, 14, 19, 18, 23, 16, 22, 17, 2, 3, 5, 4, 9, 11, 7, 6, 1, 0, 10, 8,
-        22, 22, 17, 16, 23, 13, 20, 21, 12, 15, 18, 19, 14, 7, 4, 11, 8, 2, 1, 9, 10, 5, 6, 0, 3,
-        23, 23, 16, 17, 22, 20, 13, 12, 21, 18, 15, 14, 19, 6, 5, 10, 9, 1, 2, 8, 11, 4, 7, 3, 0
-    ]
-).reshape(24, 25)[:, 1:]
+
+def get_c1_table():
```
Comment (yomach) [inline qualang_tools/bakery/randomized_benchmark_c1.py:33]: add `_` to the beginning to make it a private function (We'll move it to a different location, outside the bakery, and make it public)

File: qualang_tools/bakery/randomized_benchmark.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 from qm.qua import *
```
Comment (yomach) [inline qualang_tools/bakery/randomized_benchmark.py:2]: Undo all changed in this file.
This will be a test to make sure you didn't break backwards comptability.

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 ## [Unreleased]
 ### Fixed
 - ManualOutputControl - Fixed the `close` function
+### Changed
+- Bakery - c1_table is accessible via a function get_c1_table for compatibility with mkdocs
```
Comment (yomach) [inline CHANGELOG.md:10]: Remove this change. In the bakery we'll keep giving the `c1_table` directly.

File: qualang_tools/bakery/randomized_benchmark_c1.py [hunk 4]
```diff
@@ -29,31 +29,32 @@
 
 # Cayley table corresponding to above Clifford group structure
 
-c1_table = np.array(
```
Comment (yomach) [inline qualang_tools/bakery/randomized_benchmark_c1.py:32]: Please also leave the variable `c1_table = get_c1_table()`.
Otherwise you break backwards compatibility

File: tests/test_bakery.py [hunk 5]
```diff
@@ -6,7 +6,7 @@
 import pytest
 
 from qualang_tools.bakery.bakery import baking
-from qualang_tools.bakery.randomized_benchmark_c1 import c1_table, c1_ops
+from qualang_tools.bakery.randomized_benchmark_c1 import get_c1_table, c1_ops
```
Comment (yomach) [inline tests/test_bakery.py:9]: and same, please undo these changes

**Review by yomach — COMMENTED** [review]
Are we sure that the `__init__.py` file is supposed to be empty?

**Comment by SatyaBade12** [comment]
> Are we sure that the `__init__.py` file is supposed to be empty?

mkdocs requires en empty `__init__.py` file in the server module, see  https://www.mkdocs.org/dev-guide/themes/#:~:text=It%20must%20also%20include

**Comment by yomach** [comment]
> > Are we sure that the `__init__.py` file is supposed to be empty?
> 
> mkdocs requires en empty `__init__.py` file in the server module, see https://www.mkdocs.org/dev-guide/themes/#:~:text=It%20must%20also%20include

Very weird, sounds highly not useable if it does not support non empty init files...

**Comment by yomach** [comment]
@SatyaBade12 - Please don't force push changes after opening a PR, this complicated the PR process (for the reviewer).

**Review by yomach — APPROVED** [review]

**Comment by SatyaBade12** [comment]
> > > Are we sure that the `__init__.py` file is supposed to be empty?
> > 
> > 
> > mkdocs requires en empty `__init__.py` file in the server module, see https://www.mkdocs.org/dev-guide/themes/#:~:text=It%20must%20also%20include
> 
> Very weird, sounds highly not useable if it does not support non empty init files...

it seems that the __init__.py doesn't have to be empty (see for e.g. init file in control_panel submodule, mkdocs has no issue with it), we only need the file to be present in the directory.

**Comment by SatyaBade12** [comment]
> @SatyaBade12 - Please don't force push changes after opening a PR, this complicated the PR process (for the reviewer).

Sure, if you prefer. I didn't see a reason to break into multiple commits, at the end it is a very minor change (you can also click on the last pushed commit to see the final changes)
